network-area-diagram-viewer/src/main/resources/mainView.fxml

```diff
@@ -114,6 +114,7 @@
                     </Separator>
                     <GridPane hgap="5" vgap="5">
                         <Label style="-fx-font-weight: bold" text="SVG parameters:" GridPane.rowIndex="0"/>
+                        <CheckBox fx:id="edgeNameDisplayed" text="Display edge name" selected="true" GridPane.rowIndex="1"/>
```

Comment:
the unused rowIndew was there on purpose to have some spacing between the title and the checkboxes below